src/Components/ClustersListTable/ClustersListTable.cy.js

```diff
@@ -258,7 +319,7 @@ describe('clusters list table', () => {
     });
 
     it('applies total risk "All clusters" filter', () => {
-      hasChip('Total risk', 'All clusters');
+      hasChip('Total risk', 'All clusters', 'cluster');
```

Comment:
Has chips should be agnostic of the page

Reply:
yeah, that was a leftover from the first version. Good catch

Reply:
@Fewwy you still have this code of `hasChip` with 3 parameters in many other places

Reply:
Removed them everywhere